espresso/tools/Makefile

```
@@ -79,3 +80,8 @@ pychain:
 	export PATH=$(PYTHON_DIR):$$PATH && \
 	cd pychain/openfst_binding && python3 setup.py install && \
 	cd ../pytorch_binding && python3 setup.py install
+
+.PHONY: lhotse
+lhotse:
+	test -d lhotse || git clone https://github.com/lhotse-speech/lhotse.git
+	export PATH=$(PYTHON_DIR):$$PATH && cd lhotse && pip install -e '.[dev]'
```

```suggestion
	export PATH=$(PYTHON_DIR):$$PATH && cd lhotse && pip install -e .
```

I don't think you need a `dev` installation here? That would also install lots of packages used for building documentation, etc. And if you want to be able to run tests, you can get away with `.[tests]`.